aavealarm/Settings.tsx

```diff
@@ -1,15 +1,34 @@
 import { Slider } from "@miblanchard/react-native-slider";
 import { useEffect, useState } from "react";
-import { Platform, Switch, Text, View } from "react-native";
+import { Platform, Switch, Text, TextInput, View } from "react-native";
 import LinearGradient from "react-native-linear-gradient";
 import { getSupabase } from "./supabase";
 import * as SecureStore from "expo-secure-store";
 import { BalancesSettings, BalancesSettingsContextType, useBalancesSettings } from "./Contexts/BalancesSettings";
 
 const MAX_HEALTH_FACTOR = 10;
 
+type SettingsForm = {
+  sliderValue: number | null;
```

Comment:
I am not sure that I get the reason for these 2 new fields. Maybe we could keep only one field?